app/assets/javascripts/global.js.coffee

```diff
@@ -0,0 +1,42 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).on "page:change", -> 
+	$('#all-link').click -> 
+		$('#patients-section').fadeToggle()
+
+root = exports ? this
+
+root.showWhenValueEquals = (selector, toggleValue, toggleSelector, action = 'change') ->
+  if $(selector).prop('tagName') is 'SELECT'
+    showWhenValueEqualsOptions selector, toggleValue, toggleSelector, action
+  else
+    showWhenValueEqualsChecked selector, toggleValue, toggleSelector, action
+
+showWhenValueEqualsChecked = (selector, toggleValue, toggleSelector, action = 'change') ->
+  $(selector).on action, ->
+    selected = this
+    exemptionField = $(toggleSelector)
```

Comment:
These two functions `showWhenValueEqualsChecked` and `showWhenValueEqualsOptions` can be collapsed. Can you add a TODO assigned to me to collapse?